dags/deletes.py

```diff
@@ -292,12 +292,26 @@ def count_pending_deletes(client: Client) -> int:
 
 @op
 def wait_for_delete_mutations(
+    context: OpExecutionContext,
     cluster: ResourceParam[ClickhouseCluster],
     delete_person_events: tuple[PendingPersonEventDeletesTable, ShardMutations],
 ) -> PendingPersonEventDeletesTable:
     pending_person_deletions, shard_mutations = delete_person_events
 
-    cluster.map_all_hosts_in_shards([(shard, mutation.wait) for shard, mutation in shard_mutations.items()]).result()
+    results = cluster.map_all_hosts_in_shards(
+        {shard: mutation.wait for shard, mutation in shard_mutations.items()}
+    ).as_completed()
+    try:
+        for shard, result in results:
+            result.result()
+            context.log.info(
+                f"Shard {shard.shard_num if shard.shard_num else 'unknown'} replica {shard.replica_num if shard.replica_num else 'unknown'} completed "
+            )
```

Comment:
Something similar will already be logged with host info and shard number by default by the cluster client, although at DEBUG level - maybe we should just increase it there